docs/FAQ/FAQ-LightningNetwork.md

```diff
@@ -317,6 +318,15 @@ You can change 40 to whatever number you wish. The number represents the lines o
 
 `/var/lib/docker/volumes/generated_lnd_bitcoin_datadir/_data`
 
+### Which macaroon needs to be provided for external nodes?
```

Comment:
Needs to be added to FAQ-Readme as well😉

Reply:
Thanks boss! 👍